packages/docs/src/page-configs/ui-elements/navbar/examples/Colors.vue

```diff
@@ -1,49 +1,59 @@
 <template>
```

Comment:
<img width="641" alt="Screenshot 2022-11-24 at 14 55 14" src="https://user-images.githubusercontent.com/46869767/203778678-bbbf6d3c-1360-47cf-96ea-120bea0b4894.png">
Maybe we cna change black backgound to something else when turn on dark theme?

Reply:
@xx13

Reply:
in dark theme, let's tweak the 3rd example bg color to `#FBCAF6`, and logo + elements color to `#481269` to make it obvious you can customise everything.